### PR TITLE
IOTEDGE-1183 add Postman collection for things endpoint

### PR DESCRIPTION
**Pull request by LimaEchoCharlie**

### CODE REVIEW

File: docs/things-endpoint.md [hunk 1]
```diff
@@ -206,6 +208,9 @@ To obtain an new User Token by exchanging a Refresh Token, perform an HTTP POST
     ```
 
 1. Install and configure AM as described in the [IoT evaluation guide](https://backstage.forgerock.com/docs/iot/7.1/evaluation-guide/before-you-start.html#install-am).
+1. Modify the IoT OAuth 2.0 Client `forgerock-iot-oauth2-client`:
+    * Add `publish` scope to `Scope(s)`.
+    * Add the `Refresh Token` grant to `Grant Types`.
```
Comment (JacoJooste) [inline docs/things-endpoint.md:213]: As far as I can see the refresh token grant type is already added by the IoT Service in AM, so if the IoT evaluation guide was followed then this step should not be required.

File: docs/things-endpoint.md [hunk 2]
```diff
@@ -406,6 +411,9 @@ echo "$introspection" | jq '.'
     * curl
     * [jq](https://stedolan.github.io/jq/)
 1. Install and configure AM as described in the [IoT evaluation guide](https://backstage.forgerock.com/docs/iot/7.1/evaluation-guide/before-you-start.html#install-am).
+1. Modify the IoT OAuth 2.0 Client `forgerock-iot-oauth2-client`:
+    * Add `publish` scope to `Scope(s)`.
+    * Add the `Refresh Token` grant to `Grant Types`.
```
Comment (JacoJooste) [inline docs/things-endpoint.md:416]: Same comment as above.

File: docs/things.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,539 @@
...
+		"name": "things",
+		"description": "## Overview\n\nAM provides REST APIs under`/json/things/*` for the following use cases:\n\n* Get the attributes of a Thing\n* Obtain an OAuth 2.0 Access Token\n* Token Introspection\n* Obtain an OAuth 2.0 User Code\n* Obtain an OAuth 2.0 User Token\n* Refresh an OAuth 2.0 User Token\n\nTo use the endpoint, a Thing must be in procession of a valid session token (SSO Token). How a request to AM is constructed is dependant on whether the SSO token it has received from AM has a JWT PoP restriction. The following collection uses a simple username and password to authenticate so the simpler unrestricted calls are shown.\n\n## Prerequisites \n\nTo run this collection: \n\n* Install and configure AM as described in the [IoT evaluation guide](https://backstage.forgerock.com/docs/iot/7.1/evaluation-guide/before-you-start.html#install-am).\n* Modify the IoT OAuth 2.0 Client `forgerock-iot-oauth2-client`:\n    * Add `publish` scope to `Scope(s)`.\n    * Add the `Refresh Token` grant to `Grant Types`.\n* Set the collection variables `amURL` to the base URL of your AM instance e.g. `http://am.localtest.me:8080/openam`.",
```
Comment (JacoJooste) [inline docs/things.postman_collection.json:5]:
```suggestion
		"description": "## Overview\n\nAM provides REST APIs under`/json/things/*` for the following use cases:\n\n* Get the attributes of a Thing\n* Obtain an OAuth 2.0 Access Token\n* Token Introspection\n* Obtain an OAuth 2.0 User Code\n* Obtain an OAuth 2.0 User Token\n* Refresh an OAuth 2.0 User Token\n\nTo use the endpoint, a Thing must be in possession of a valid session token (SSO Token). How a request to AM is constructed is dependant on whether the SSO token it has received from AM has a JWT PoP restriction. The following collection uses a simple username and password to authenticate so the simpler unrestricted calls are shown.\n\n## Prerequisites \n\nTo run this collection: \n\n* Install and configure AM as described in the [IoT evaluation guide](https://backstage.forgerock.com/docs/iot/7.1/evaluation-guide/before-you-start.html#install-am).\n* Modify the IoT OAuth 2.0 Client `forgerock-iot-oauth2-client`:\n    * Add `publish` scope to `Scope(s)`.\n    * Set the collection variables `amURL` to the base URL of your AM instance e.g. `http://am.localtest.me:8080/openam`.",
```

File: docs/things.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,539 @@
+{
+	"info": {
+		"_postman_id": "2436794b-3a84-4c13-8b92-9f6ee5f24f8d",
+		"name": "things",
```
Comment (JacoJooste) [inline docs/things.postman_collection.json:4]:
```suggestion
		"name": "ForgeRock IoT Collection",
```
This is a bit more consistent with our ID Cloud name: https://backstage.forgerock.com/docs/idcloud/latest/_attachments/ForgeRock_Identity_Cloud.postman_collection.json

**Review by JacoJooste — APPROVED** [review]
This is great James 👍🏻

File: docs/things-endpoint.md (hunk 1 above)
Comment (LimaEchoCharlie) [reply docs/things-endpoint.md:213]: Great spot. Furthermore, the `publish` scope is added in the evaluation guide so this section can be removed. I'll also update the video.

**Pull request merged**